Bottom: 703fa5cee0ffd57a5871914a938d05d7d9193190
Top:    943b18cf655bb6c3b7d8ffacd06d89687a5e9d04
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-19 01:01:38 +0000

Refresh of push-special-case-thindraw-vs-

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 6c6f26c..af719fd 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -484,7 +484,6 @@ void ghid_draw_rect (hidGC gc, Coord x1, Coord y1, Coord x2, Coord y2);
 void ghid_fill_circle (hidGC gc, Coord cx, Coord cy, Coord radius);
 void ghid_fill_polygon (hidGC gc, int n_coords, Coord *x, Coord *y);
 void ghid_fill_rect (hidGC gc, Coord x1, Coord y1, Coord x2, Coord y2);
-void ghid_draw_pcb_polygon (hidGC gc, PolygonType *polygon, const BoxType *drawn_area);
 void ghid_invalidate_lr (int left, int right, int top, int bottom);
 void ghid_invalidate_all ();
 void ghid_notify_crosshair_change (bool changes_complete);
